config/spinnaker.yml

```diff
@@ -231,8 +231,8 @@ services:
 
   stackdriver:
     enabled: ${SPINNAKER_STACKDRIVER_ENABLED:false}
-    projectName: ${SPINNAKER_STACKDRIVER_PROJECT_NAME:${providers.google.primaryCredentials.project}}
-    credentialsPath: ${SPINNAKER_STACKDRIVER_CREDENTIALS_PATH:${providers.google.primaryCredentials.jsonPath}}   
+    projectName: ${providers.google.primaryCredentials.project}
+    credentialsPath: ${providers.google.primaryCredentials.jsonPath} 
 
```

Comment:
I'm back at my desk now and playing around with all the different options - I'm guess the above won't work because it will resolve to that string literal `${...}` rather than the expansion of the expression, but I'll try that one too.